ajax/libs/bulmaswatch/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bulmaswatch",
+  "description": "Themes for Bulma",
+  "author": "Jenil Gogari <i@jgog.in>",
+  "homepage": "https://jenil.github.io/bulmaswatch",
+  "filename": "bulmaswatch.min.css",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jenil/bulmaswatch.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "css",
+    "sass",
+    "flexbox",
+    "responsive",
+    "framework",
+    "themes"
+  ],
+  "autoupdate": 
+  {
+    "source": "git",
+    "target": "git://github.com/jenil/bulmaswatch.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "bulmaswatch*.+(js|css|map)"
```

Comment:
@sufuf3 @hare1039 I think we can't have `default` in the `basePath` since each theme has its own `bulmaswatch.min.css`. I had set the `filename` assuming that it is just a default file that I need to refer.